populate/src/main/resources/seed/portals/ourhealth/participants/jsalk.json

```diff
@@ -9,10 +9,10 @@
     "sexAtBirth": "male",
     "mailingAddress": {
       "street1": "32 Elm Street",
-      "city": "New York",
-      "state": "NY",
+      "city": "Cambridge",
+      "state": "MA",
       "country": "US",
-      "postalCode": "10005"
+      "postalCode": "02142"
```

Comment:
Updating `JSALK` to have a real, valid mailing address. This isn't strictly necessary for this PR, but will be helpful for testing live integration with Pepper since Pepper will reject invalid addresses.

Hopefully this change won't break anything (other than a test that I already updated). However, it won't update the demo server database unless we re-populate which we probably don't want to do wholesale, so I'm not sure of the best way to handle that.